src/pybind/mgr/dashboard/run-backend-api-tests.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+set -e
```

Comment:
```suggestion
set -euo pipefail
```
Don't suggest the `set -euxo pipefail` because the `-x` flag is better to be enabled on demand.

Reply:
Done.

Reply:
@epuertat I finally deleted this line because @callithea detected an odd behaviour causing ths script to stop when running on CentOS 7 (confirmed by me, previously I tested it with Fedora/RHEL8).